pkg/jobrunaggregator/jobrunaggregatorlib/spyglass_summary.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
 )
 
-func htmlForJobRuns(ctx context.Context, finishedJobsToAggregate, unfinishedJobsToAggregate []jobrunaggregatorapi.JobRunInfo) string {
+func htmlForJobRuns(ctx context.Context, finishedJobsToAggregate, unfinishedJobsToAggregate []jobrunaggregatorapi.JobRunInfo, jobSummaryInfo string) string {
 	html := `<!DOCTYPE html>
 <html>
 <head>
```

Comment:
The <title> tag I mentioned would be injected here, and spyglass would use that instead